lassen-build-and-test.yml

```diff
@@ -91,54 +91,38 @@ status_pending:
     - export pipeline_status="pending"
     - *status_report
 
-# Gitlab jobs for the lassen machine at ${GITHUB_PROJECT_ORG}.
 
-# CPU only.
-ibm_clang_9_0_0:
-  variables:
-    SPEC: "${PROJECT_LASSEN_VARIANTS} %clang@ibm.9.0.0 ${PROJECT_LASSEN_DEPS}"
-  extends: .build_and_test_on_lassen
-
-ibm_clang_9_0_0_gcc_8_3_1:
-  variables:
-    SPEC: "${PROJECT_LASSEN_VARIANTS} %clang@ibm.9.0.0 cxxflags==\"--gcc-toolchain=/usr/tce/packages/gcc/gcc-8.3.1\" cflags==\"--gcc-toolchain=/usr/tce/packages/gcc/gcc-8.3.1\" ${PROJECT_LASSEN_DEPS}"
-  extends: .build_and_test_on_lassen
-
-gcc_8_3_1:
+# CUDA.
+clang_12_0_1_gcc_8_3_1_cuda_10_1_243:
```

Comment:
This one is here because it is the system default, correct?

Reply:
Yes, to quote #15:
`clang/12.0.1 w/ gcc 8.3.1 toolchain + cuda/10.1.243 (old, but system default and is being used)`